src/routes.js

```diff
@@ -139,9 +127,8 @@ const router = async app => {
 			);
 
 			// Filter out any repository that we don't manage, and map to just the name property.
-			const filtered = repositories
+			const filtered = repositories.list()
 				.filter(r => results.includes(r.id))
-				.map(({ name }) => ({ name }));
```

Comment:
TODO: Write a test to see if I've broken something here

Reply:
If all the other tests are OK, I _think_ you might be good.

We needed more than `name` previously as we were adding a removing repositories by their ID.